### ISSUE TITLE
[V3] 'Play' event isn't called after unpausing

### ISSUE THREAD

**Issue by TunnelBlanket** [sampotts/plyr#784]
The `pause` event isn't fired after resuming the player.

- [x] Issue does not already exist
- [x] Issue observed on https://plyr.io/beta

### Expected behaviour 
Firing of the `play` event when playback is resumed.

### Actual behaviour
The `playing` event is fired instead.

### Environment

- Browser: Chromium
- Version:  64
- Operating System: Windows
- Version: 10 (1607)

Players affected:
- [ ] HTML5 Video
- [ ] HTML5 Audio
- [x] YouTube
- [ ] Vimeo

**Comment by TunnelBlanket** [comment]
The reason this is a problem to me is because seeking the player **also** fires the `playing` event, so I can't use it to toggle playback.

**Comment by sampotts** [comment]
This is a weird once since all Plyr does is bubble the media elements events (usually they don't bubble).

**Comment by friday** [comment]
This is the output I get when opening the console at https://plyr.io/beta/ then pressing play, pause and play again:
```
plyr.js:271 event: play
plyr.js:271 event: playing
plyr.js:271 event: controlshidden
5 plyr.js:271 event: timeupdate
plyr.js:271 event: timeupdate
plyr.js:271 event: controlsshown
plyr.js:271 event: pause
plyr.js:271 event: play
plyr.js:271 event: playing
plyr.js:271 event: controlshidden
2 plyr.js:271 event: timeupdate
plyr.js:271 event: timeupdate
plyr.js:271 event: controlsshown
plyr.js:271 event: pause
```

Seems to work fine (Chrome 63)

**Comment by friday** [comment]
Also see https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Media_events for documentation. I don't think it's possible that Plyr or the browser is getting this wrong, so most likely the issue is invalid.

**Comment by TunnelBlanket** [comment]
Okay so, I did some more testing and this issue only seems to happen with YouTube sources. I tried setting an `video/mp4` source and the event was fired and working as expected.

**Comment by TunnelBlanket** [comment]
I confirmed this on https://plyr.io/beta after setting the source to a YouTube video.
```javascript
player.source = {type: 'video', sources: [{src: 'OTHHeIAYfuU', provider: 'youtube'}]}
```

**Comment by sampotts** [comment]
Thanks Arthur, I’ll take a look today. 

Sam

**Comment by friday** [comment]
Seems legit. I can duplicate this. https://plyr.io/beta doesn't get the "play" event after pausing when switching source to youtube, but https://plyr.io/#youtube does.

**Comment by sampotts** [comment]
I've fixed this in `v3.0.0-beta.16` which is on NPM and in releases
